manifests/0000_50_cluster-monitoring-operator_00_0alertmanager-config-custom-resource-definition.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
     controller-gen.kubebuilder.io/version: v0.2.4
     include.release.openshift.io/ibm-cloud-managed: "true"
     include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
```

Comment:
there's something not right in our tooling because this file (and the other CRDs) are generated from Jsonnet. The new annotation should be added to `jsonnet/add-release-annotation.libsonnet`. I'm investigating why the CI didn't fail.